pravega/pravega-operator
# Issue 661: Removing the check of controller minimum replicas from operator

### Change log description

Adding the support additional port in segment store service

### Purpose of the change

 Fixes #661

### What the code does
Removed the check of mandating controller replica to 1
### How to verify it
Removed the check of mandating controller replica to 1
currently the check for minimum controller replicas is removed, because during LTS recovery we need to stop all the requests from controller. In order to achieve this we are allowing `0` controller replicas.
